Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/cyp-test-nic-simple
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/cyp-test-nic-simple/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/cyp-test-nic-simple/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (set args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/conf/change_args.py '{"bmc_conf":{"user":"debuguser","password":"0penBmc1"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-21.02/","sut_conf":{"bmc_ip":"10.239.183.128","board_type":"commercial","host":"10.239.182.209","pf1":"ens785f2","os":"RHEL","PF1_NIC_type":"4*25"},"pkg_generator":{"host":"10.239.182.79","pf1":"ens787f2"},"drive_info":{"drive_version":"1.6.4"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics -s -m nicSimple --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_simple_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 19 items / 11 deselected / 8 selected

npxtester/Tests/testNics/test_mtu.py 2022-04-11 08:53:46 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Jumbo_Packet_CVL_08_53_46/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Jumbo_Packet_CVL_08_53_46
2022-04-11 08:53:46 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-11 08:53:46 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-11 08:53:46 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-11 08:53:46 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:53:46 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:53:46 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 08:53:47 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 08:53:47 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 08:53:50 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 08:53:50 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 08:53:55 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 08:53:55 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 08:53:58 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 08:54:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:54:13 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 08:54:13 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 08:54:16 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 08:54:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:54:25 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 08:54:28 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:54:28 - sshInstance.py - [line:54] - root - INFO - ens785f2: ERROR while getting interface flags: No such device

2022-04-11 08:54:36 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 08:54:36 - sshInstance.py - [line:54] - root - INFO - Cannot get driver information: No such device

2022-04-11 08:54:41 - sshInstance.py - [line:209] - root - ERROR - Case Failed:driver version mismatch version ,please check
E2022-04-11 08:54:41 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-11 08:54:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_MTU_set_CVL_08_54_41/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_MTU_set_CVL_08_54_41
2022-04-11 08:54:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-11 08:54:41 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-11 08:54:41 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-11 08:54:41 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-11 08:54:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:54:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:54:42 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 08:54:42 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:54:42 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:54:42 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 08:54:42 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 08:54:42 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 08:54:45 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 08:54:45 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 08:54:50 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 08:54:50 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 08:54:53 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 08:55:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:55:06 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 08:55:06 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 08:55:09 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 08:55:13 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:55:18 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 08:55:21 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:55:22 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:55:30 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 08:55:30 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 08:55:38 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-11 08:55:38 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:55:46 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 2000
2022-04-11 08:55:46 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:55:54 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 08:55:54 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2204 (2.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 08:56:03 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:05:14 2022 from 10.239.182.99
[root@npx-cypsilver1-rel84 ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | Craft packets like it is your last[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   | day on earth.[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |                      -- Lao-Tze[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m[34m[1m   |[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-11 08:56:10 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*18890)],iface="ens787f2")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-11 08:56:13 - test_mtu.py - [line:132] - root - INFO - success to send package to network
2022-04-11 08:56:17 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="40:a6:b7:72:cc:ca")/IP(len=46)/Raw(load='X'*20010)],iface="ens787f2")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-11 08:56:20 - test_mtu.py - [line:142] - root - INFO - result is right
.2022-04-11 08:56:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 mtu 1500
2022-04-11 08:56:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:56:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 mtu 1500
2022-04-11 08:56:32 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:56:40 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 08:56:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:56:48 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 08:56:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:56:53 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 08:56:53 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_PI_Networking_MTU_set_CVL_08_54_41
2022-04-11 08:56:56 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 08:56:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:57:01 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 08:57:01 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-11 08:57:01 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
                                  [ 25%]
npxtester/Tests/testNics/test_nics.py 2022-04-11 08:57:01 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-11 08:57:01 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_silver_Networking_ConnectivityEnableDisable_08_57_01/test_silver_Networking_ConnectivityEnableDisable.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_silver_Networking_ConnectivityEnableDisable_08_57_01
2022-04-11 08:57:01 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-04-11 08:57:01 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 08:57:01 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:57:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:57:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 08:57:01 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 08:57:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:57:04 - sshInstance.py - [line:54] - root - INFO - ens785f2: ERROR while getting interface flags: No such device

2022-04-11 08:57:12 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 08:57:12 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 08:57:17 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 08:57:17 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 08:57:20 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 08:57:20 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 08:57:25 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 08:57:25 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 08:57:28 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 08:57:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:57:45 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 08:57:45 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 08:57:48 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 08:57:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:57:57 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 08:58:00 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:58:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:08 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 08:58:08 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 08:58:16 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 08:58:16 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:21 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 08:58:24 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 08:58:25 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 08:58:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 08:58:33 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095673814  bytes 9228840562340 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851378  bytes 17453491387 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 08:58:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 08:58:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:49 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-11 08:58:49 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:58 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:09:55 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
2022-04-11 08:59:12 - sshInstance.py - [line:75] - root - INFO - 
--- 192.168.10.10 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 89ms

[root@npx-cyp7-rh82 ~]# 
F2022-04-11 08:59:18 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 08:59:18 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:59:26 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 08:59:26 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:59:34 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 08:59:34 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:59:42 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 08:59:42 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:59:50 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 08:59:50 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:59:55 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 08:59:55 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_silver_Networking_ConnectivityEnableDisable_08_57_01
2022-04-11 08:59:58 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 08:59:58 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:00:03 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:00:03 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:00:03 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:00:03 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-04-11 09:00:03 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_silver_ConnectivityReboot_09_00_03/test_silver_ConnectivityReboot.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_silver_ConnectivityReboot_09_00_03
2022-04-11 09:00:03 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_ConnectivityReboot case ============================
2022-04-11 09:00:03 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:00:03 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:00:03 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:00:03 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:00:03 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:00:03 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:00:03 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:00:03 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:00:04 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:00:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:00:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:00:15 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:00:15 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:00:20 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:00:20 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:00:23 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:00:23 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:00:28 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:00:28 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:00:31 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:00:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:00:45 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:00:45 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:00:48 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:00:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:00:58 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:01:01 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:01:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:01:09 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:01:09 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:01:17 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:01:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:01:22 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:01:25 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:01:25 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:01:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:01:33 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095673859  bytes 9228840568786 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851389  bytes 17453494155 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:01:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-11 09:01:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:01:49 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:01:49 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:01:54 - test_nics.py - [line:138] - root - INFO - -- reboot 1 times --
2022-04-11 09:01:58 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:12:57 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.467 ms
2022-04-11 09:01:59 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.178 ms
2022-04-11 09:02:00 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.211 ms
2022-04-11 09:02:01 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.197 ms
2022-04-11 09:02:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.191 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 128ms
rtt min/avg/max/mdev = 0.178/0.248/0.467/0.111 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:02:09 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.516 ms
2022-04-11 09:02:10 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.203 ms
2022-04-11 09:02:11 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.191 ms
2022-04-11 09:02:12 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.194 ms
2022-04-11 09:02:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.198 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 123ms
rtt min/avg/max/mdev = 0.191/0.260/0.516/0.128 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:02:22 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-11 09:02:25 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-11 09:03:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 128ms
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:03:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 102ms
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:40 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:03:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 140ms
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:50 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:03:50 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:03:55 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 114ms
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:00 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:00 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:04:05 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 88ms
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:10 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:10 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:04:15 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 126ms
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:04:25 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 99ms
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:30 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:30 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:04:35 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.378 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.383 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.375 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.350 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.405 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 137ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.350/0.378/0.405/0.021 ms
2022-04-11 09:04:40 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:04:40 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:04:40 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:07:41 - test_nics.py - [line:138] - root - INFO - -- reboot 2 times --
2022-04-11 09:07:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:07:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:07:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:07:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:07:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:07:52 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:07:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:08:01 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:17:35 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.449 ms
2022-04-11 09:08:02 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.195 ms
2022-04-11 09:08:03 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.193 ms
2022-04-11 09:08:04 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.185 ms
2022-04-11 09:08:05 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.203 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 129ms
rtt min/avg/max/mdev = 0.185/0.245/0.449/0.102 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:08:12 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.484 ms
2022-04-11 09:08:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.217 ms
2022-04-11 09:08:14 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.213 ms
2022-04-11 09:08:15 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.214 ms
2022-04-11 09:08:16 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.199 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 123ms
rtt min/avg/max/mdev = 0.199/0.265/0.484/0.110 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:08:25 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-11 09:08:28 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-11 09:09:28 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 132ms
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:33 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:09:38 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 106ms
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:43 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:43 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:09:48 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 80ms
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:53 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:09:53 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:09:58 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 118ms
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:03 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:03 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:10:08 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 92ms
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:13 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:13 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:10:18 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 129ms
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:23 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:23 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:10:28 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.358 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.361 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.390 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.344 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.324 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 103ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.324/0.355/0.390/0.027 ms
2022-04-11 09:10:33 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:10:33 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:10:33 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:13:33 - test_nics.py - [line:138] - root - INFO - -- reboot 3 times --
2022-04-11 09:13:33 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:13:33 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:13:33 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:13:36 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:13:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:13:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:13:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:13:53 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:23:27 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.423 ms
2022-04-11 09:13:54 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.199 ms
2022-04-11 09:13:55 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.207 ms
2022-04-11 09:13:56 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.199 ms
2022-04-11 09:13:57 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.210 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 112ms
rtt min/avg/max/mdev = 0.199/0.247/0.423/0.089 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:14:04 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.499 ms
2022-04-11 09:14:05 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.237 ms
2022-04-11 09:14:06 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.176 ms
2022-04-11 09:14:07 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.249 ms
2022-04-11 09:14:08 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.209 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 106ms
rtt min/avg/max/mdev = 0.176/0.274/0.499/0.115 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:14:17 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-11 09:14:20 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-11 09:15:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 99ms
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:25 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:15:30 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 136ms
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:35 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:15:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 110ms
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:15:50 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 83ms
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:15:55 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:16:00 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 121ms
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:05 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:16:10 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 94ms
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:15 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:16:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.365 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.370 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.371 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.382 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.376 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 132ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.365/0.372/0.382/0.025 ms
2022-04-11 09:16:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:16:25 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:16:25 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:19:26 - test_nics.py - [line:138] - root - INFO - -- reboot 4 times --
2022-04-11 09:19:26 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:19:26 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:19:26 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:19:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:19:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:19:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:19:37 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:19:46 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:29:19 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.464 ms
2022-04-11 09:19:47 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.178 ms
2022-04-11 09:19:48 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.176 ms
2022-04-11 09:19:49 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.182 ms
2022-04-11 09:19:50 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.200 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 139ms
rtt min/avg/max/mdev = 0.176/0.240/0.464/0.112 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:19:57 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.501 ms
2022-04-11 09:19:58 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.216 ms
2022-04-11 09:19:59 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.198 ms
2022-04-11 09:20:00 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.186 ms
2022-04-11 09:20:01 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.188 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 132ms
rtt min/avg/max/mdev = 0.186/0.257/0.501/0.123 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:20:10 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-11 09:20:13 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-11 09:21:13 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 128ms
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:18 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:21:23 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 101ms
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:28 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:21:33 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 139ms
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:38 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:21:43 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 113ms
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:48 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:21:53 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 87ms
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:21:58 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:03 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 124ms
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:08 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:13 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 98ms
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:18 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:23 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 135ms
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:28 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:33 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 109ms
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:38 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:43 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 82ms
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:48 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:22:53 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 120ms
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:22:58 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:03 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 94ms
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:08 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:13 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 132ms
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:18 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:23 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 106ms
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:28 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:33 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 80ms
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:38 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:43 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 118ms
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:48 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:23:53 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 92ms
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:23:58 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:03 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 129ms
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:08 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:13 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 103ms
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:18 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:23 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 77ms
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:28 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:33 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 114ms
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:38 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:43 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 89ms
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:48 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:24:53 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 126ms
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:24:58 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:03 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 100ms
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:08 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:13 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 137ms
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:18 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:18 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:23 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 110ms
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:28 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:28 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:33 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 84ms
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:38 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:38 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:43 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 121ms
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:48 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:48 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:25:53 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 95ms
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:58 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:25:58 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:26:03 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.351 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.377 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.389 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.381 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.282 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 133ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.282/0.356/0.389/0.039 ms
2022-04-11 09:26:08 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:26:08 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:26:08 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:29:08 - test_nics.py - [line:138] - root - INFO - -- reboot 5 times --
2022-04-11 09:29:08 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:29:08 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:29:08 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:29:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:29:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:29:19 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:29:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:29:28 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:39:00 2022
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.11
PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.457 ms
2022-04-11 09:29:29 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.203 ms
2022-04-11 09:29:30 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.190 ms
2022-04-11 09:29:31 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.193 ms
2022-04-11 09:29:32 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.196 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 140ms
rtt min/avg/max/mdev = 0.190/0.247/0.457/0.106 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:29:39 - sshInstance.py - [line:75] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.513 ms
2022-04-11 09:29:40 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.195 ms
2022-04-11 09:29:41 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.206 ms
2022-04-11 09:29:42 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.194 ms
2022-04-11 09:29:43 - sshInstance.py - [line:75] - root - INFO - 64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.213 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 133ms
rtt min/avg/max/mdev = 0.194/0.264/0.513/0.125 ms
[root@npx-cyp7-rh82 ~]# 
2022-04-11 09:29:52 - sshInstance.py - [line:99] - root - INFO - reboot

2022-04-11 09:29:55 - sshInstance.py - [line:159] - root - INFO - sut server reboot, please wait ......
2022-04-11 09:30:56 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 129ms
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:01 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:01 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:06 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 103ms
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:11 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:11 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:16 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 78ms
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:21 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:21 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:26 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 116ms
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:31 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:31 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:36 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 89ms
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:41 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:41 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:46 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 127ms
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:51 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:31:51 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:31:56 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.345 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.385 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.365 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.341 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.314 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 101ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.314/0.350/0.385/0.023 ms
2022-04-11 09:32:01 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:32:01 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:32:01 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
.2022-04-11 09:35:04 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:35:04 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:35:04 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:35:04 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:35:04 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:35:07 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:35:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:15 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 09:35:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 09:35:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:31 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 09:35:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:36 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 09:35:36 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_silver_ConnectivityReboot_09_00_03
2022-04-11 09:35:39 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 09:35:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:44 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:35:44 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:35:44 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:35:44 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_silver_ConnectivityReboot case ============================
2022-04-11 09:35:44 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_gold_PI_Networking_SRIOV_CheckDevice_09_35_44/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_gold_PI_Networking_SRIOV_CheckDevice_09_35_44
2022-04-11 09:35:44 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-11 09:35:44 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:35:44 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:35:44 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:35:45 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:35:45 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:35:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:35:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:56 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:35:56 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:36:01 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:36:01 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:36:04 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:36:04 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:36:09 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:36:09 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:36:12 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:36:24 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:29 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:36:29 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:36:32 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:36:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:41 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:36:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:36:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:36:52 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:37:00 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:37:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:05 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:37:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:37:08 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:37:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:37:16 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095674227  bytes 9228840622489 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851536  bytes 17453517941 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:37:24 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:37:25 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:38 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11

2022-04-11 09:37:38 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 down
2022-04-11 09:37:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 up
2022-04-11 09:37:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 192.168.10.12
2022-04-11 09:38:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:15 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-11 09:38:19 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.405 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.194 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.162 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.207 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.210 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4087ms
rtt min/avg/max/mdev = 0.162/0.235/0.405/0.088 ms

2022-04-11 09:38:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 down
2022-04-11 09:38:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 up
2022-04-11 09:38:43 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 192.168.10.13
2022-04-11 09:38:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:59 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.13
2022-04-11 09:39:06 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4110ms
pipe 4

F2022-04-11 09:39:17 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:39:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:39:28 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:39:28 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:39:36 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 09:39:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:39:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 09:39:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:39:52 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 09:39:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:39:57 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 09:39:57 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_gold_PI_Networking_SRIOV_CheckDevice_09_35_44
2022-04-11 09:40:00 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 09:40:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:40:05 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:40:05 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:40:05 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:40:05 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-11 09:40:05 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_InternetProtocolv4v6_09_40_05/test_PI_Networking_InternetProtocolv4v6.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_InternetProtocolv4v6_09_40_05
2022-04-11 09:40:05 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-04-11 09:40:05 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:40:05 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:40:05 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:40:05 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:40:05 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:40:05 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:40:05 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:40:06 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:40:06 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:40:09 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:40:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:40:17 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:40:17 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:40:22 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:40:22 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:40:25 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:40:25 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:40:30 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:40:30 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:40:33 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:40:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:40:49 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:40:49 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:40:52 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:40:57 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:41:02 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:41:05 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:41:05 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:41:13 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:41:13 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:41:21 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:41:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:41:26 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:41:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:41:29 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:41:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:41:37 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095674465  bytes 9228840660103 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851571  bytes 17453523919 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:41:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:41:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:41:53 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-11 09:41:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:42:01 - sshInstance.py - [line:50] - root - INFO - ping -w 5 192.168.10.11
2022-04-11 09:42:06 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.437 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.197 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.202 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.187 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.202 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 122ms
rtt min/avg/max/mdev = 0.187/0.245/0.437/0.096 ms

2022-04-11 09:42:14 - sshInstance.py - [line:50] - root - INFO - ping6 -c 5 -I ens785f2 fe80::42a6:b7ff:fe82:eb6
2022-04-11 09:42:19 - sshInstance.py - [line:54] - root - INFO - ping6: Warning: source address might be selected on device other than ens785f2.
PING fe80::42a6:b7ff:fe82:eb6(fe80::42a6:b7ff:fe82:eb6) from :: ens785f2: 56 data bytes
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=1 ttl=64 time=0.495 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=2 ttl=64 time=0.172 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=3 ttl=64 time=0.189 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=4 ttl=64 time=0.186 ms
64 bytes from fe80::42a6:b7ff:fe82:eb6%ens785f2: icmp_seq=5 ttl=64 time=0.215 ms

--- fe80::42a6:b7ff:fe82:eb6 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 134ms
rtt min/avg/max/mdev = 0.172/0.251/0.495/0.123 ms

.2022-04-11 09:42:27 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:42:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:42:35 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:42:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:42:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 09:42:43 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:42:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 09:42:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:42:59 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 09:42:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:43:04 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 09:43:04 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_PI_Networking_InternetProtocolv4v6_09_40_05
2022-04-11 09:43:07 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 09:43:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:43:12 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:43:12 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:43:12 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:43:12 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_InternetProtocolv4v6 case ============================
2022-04-11 09:43:12 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Link_State_Check_09_43_12/test_PI_Networking_Link_State_Check.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Link_State_Check_09_43_12
2022-04-11 09:43:12 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Link_State_Check case ============================
2022-04-11 09:43:12 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:43:12 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:43:12 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:43:12 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:43:12 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:43:12 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:43:12 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:43:12 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:43:12 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:43:15 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:43:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:43:23 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:43:23 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:43:29 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:43:29 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:43:32 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:43:32 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:43:37 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:43:37 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:43:40 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:43:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:43:53 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:43:53 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:43:56 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:44:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:44:05 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:44:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:44:09 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:44:17 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:44:17 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:44:25 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:44:25 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:44:30 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:44:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:44:33 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 342 (342.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:44:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:44:41 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095674511  bytes 9228840666059 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851606  bytes 17453530505 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:44:49 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-21.02/usertools/dpdk-devbind.py -s | grep ens785f2 | awk '{print $1;}'
2022-04-11 09:44:49 - sshInstance.py - [line:54] - root - INFO - 0000:4b:00.2

2022-04-11 09:44:57 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:4b:00.2 | grep LnkCap:
2022-04-11 09:44:58 - sshInstance.py - [line:54] - root - INFO - 		LnkCap:	Port #0, Speed 16GT/s, Width x16, ASPM not supported

2022-04-11 09:45:06 - sshInstance.py - [line:50] - root - INFO - lspci -vvs 00:4b:00.2 | grep LnkSta:
2022-04-11 09:45:07 - sshInstance.py - [line:54] - root - INFO - 		LnkSta:	Speed 16GT/s, Width x16, TrErr- Train- SlotClk+ DLActive- BWMgmt- ABWMgmt-

2022-04-11 09:45:12 - test_nics.py - [line:283] - root - INFO - --check the speed and width of the ethernet ens785f2
2022-04-11 09:45:12 - test_nics.py - [line:287] - root - INFO - the linkCap and LnkSta value are matched. The speed and width value are: 16GT/s and Width x16.
.2022-04-11 09:45:15 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:45:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:45:23 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:45:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:45:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 09:45:31 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:45:39 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 09:45:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:45:47 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 09:45:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:45:52 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 09:45:52 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Link_State_Check_09_43_12
2022-04-11 09:45:55 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 09:45:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:46:01 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:46:01 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:46:01 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:46:01 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Link_State_Check case ============================
2022-04-11 09:46:01 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_ConnectivityShutdown_09_46_01/test_PI_Networking_ConnectivityShutdown.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_ConnectivityShutdown_09_46_01
2022-04-11 09:46:01 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_ConnectivityShutdown case ============================
2022-04-11 09:46:01 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:46:01 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:46:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:46:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:46:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:46:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:46:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:46:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:46:01 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:46:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:46:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:46:12 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:46:12 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:46:17 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:46:17 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:46:20 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:46:20 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:46:25 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:46:25 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:46:28 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:46:38 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:46:43 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:46:43 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:46:46 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:46:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:46:56 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:46:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:46:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:47:07 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:47:07 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:47:15 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:47:15 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:47:20 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:47:23 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:47:23 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 16  bytes 2134 (2.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:47:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:47:31 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095674532  bytes 9228840668763 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851608  bytes 17453530759 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:47:39 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-11 09:47:39 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:47:47 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:47:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:47:55 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-04-11 09:47:59 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.439 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.221 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.203 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.207 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.212 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 84ms
rtt min/avg/max/mdev = 0.203/0.256/0.439/0.092 ms

2022-04-11 09:48:04 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-11 09:48:04 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-11 09:48:04 - commonFunc.py - [line:30] - root - INFO - Chassis Power is on
2022-04-11 09:48:04 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:48:04 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-11 09:48:04 - power.py - [line:22] - root - INFO - Chassis Power is on

2022-04-11 09:48:04 - test_nics.py - [line:328] - root - INFO - Chassis Power is on

2022-04-11 09:48:04 - sshInstance.py - [line:143] - root - INFO - ping sut, please wait ......
2022-04-11 09:49:04 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.315 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.376 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.334 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.367 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.380 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 86ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.315/0.354/0.380/0.030 ms
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:49:09 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:49:09 - power.py - [line:39] - root - INFO - starting to power off the machine....
2022-04-11 09:49:09 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power off
....
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - Chassis Power Control: Down/Off
2022-04-11 09:49:09 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:49:09 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power off

2022-04-11 09:49:09 - power.py - [line:44] - root - INFO - Chassis Power Control: Down/Off

2022-04-11 09:49:09 - test_nics.py - [line:331] - root - INFO - Chassis Power Control: Down/Off

2022-04-11 09:49:19 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-11 09:49:19 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-11 09:49:20 - commonFunc.py - [line:30] - root - INFO - Chassis Power is off
2022-04-11 09:49:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:49:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-11 09:49:20 - power.py - [line:22] - root - INFO - Chassis Power is off

2022-04-11 09:49:20 - test_nics.py - [line:335] - root - INFO - the machine is powering off....
2022-04-11 09:49:20 - power.py - [line:27] - root - INFO - starting to power on the machine....
2022-04-11 09:49:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power on
....
2022-04-11 09:49:20 - commonFunc.py - [line:30] - root - INFO - Chassis Power Control: Up/On
2022-04-11 09:49:20 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:49:20 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power on

2022-04-11 09:49:20 - power.py - [line:32] - root - INFO - Chassis Power Control: Up/On

2022-04-11 09:49:20 - test_nics.py - [line:341] - root - INFO - Chassis Power Control: Up/On

2022-04-11 09:49:30 - sshInstance.py - [line:143] - root - INFO - ping sut, please wait ......
2022-04-11 09:50:30 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 100ms
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:35 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:50:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 138ms
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:50:50 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 111ms
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:50:55 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:00 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 85ms
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:05 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:10 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 122ms
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:15 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 96ms
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:25 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:30 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 133ms
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:35 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 107ms
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:51:50 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 80ms
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:55 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:51:55 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:52:00 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 117ms
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:05 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:05 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:52:10 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 91ms
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:15 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:15 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:52:20 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 129ms
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:25 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:25 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:52:30 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 0 received, 100% packet loss, time 101ms
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:35 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:35 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:52:40 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ping -w 5 10.239.182.209....
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - PING 10.239.182.209 (10.239.182.209) 56(84) bytes of data.
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=1 ttl=61 time=0.287 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=2 ttl=61 time=0.371 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=3 ttl=61 time=0.364 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=4 ttl=61 time=0.344 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 64 bytes from 10.239.182.209: icmp_seq=5 ttl=61 time=0.397 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - --- 10.239.182.209 ping statistics ---
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 5 packets transmitted, 5 received, 0% packet loss, time 139ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - rtt min/avg/max/mdev = 0.287/0.352/0.397/0.042 ms
2022-04-11 09:52:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:52:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ping -w 5 10.239.182.209
2022-04-11 09:57:45 - power.py - [line:18] - root - INFO - checking the machine status
2022-04-11 09:57:45 - commonFunc.py - [line:16] - root - INFO - starting to execute the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status
....
2022-04-11 09:57:45 - commonFunc.py - [line:30] - root - INFO - Chassis Power is on
2022-04-11 09:57:45 - commonFunc.py - [line:30] - root - INFO - 
2022-04-11 09:57:45 - commonFunc.py - [line:31] - root - INFO - finished exeucting the cmd: ipmitool -I lanplus -H 10.239.183.128 -U debuguser -P 0penBmc1 power status

2022-04-11 09:57:45 - power.py - [line:22] - root - INFO - Chassis Power is on

2022-04-11 09:57:45 - test_nics.py - [line:347] - root - INFO - the machine is powering on....
2022-04-11 09:57:45 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:57:45 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:57:45 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:57:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 09:57:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:57:56 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.11

2022-04-11 09:58:00 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.11 (192.168.10.11) 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.447 ms
64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.194 ms
64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.191 ms
64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.191 ms
64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.192 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 80ms
rtt min/avg/max/mdev = 0.191/0.243/0.447/0.102 ms

.2022-04-11 09:58:08 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:58:08 - transport.py - [line:1819] - paramiko.transport - ERROR - Socket exception: Connection reset by peer (104)
2022-04-11 09:58:08 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:58:09 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:58:09 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:58:09 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:58:12 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:58:12 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:58:20 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 0
2022-04-11 09:58:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:58:28 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 0
2022-04-11 09:58:28 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:58:36 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-11 09:58:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:58:41 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-11 09:58:41 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-11/test_PI_Networking_ConnectivityShutdown_09_46_01
2022-04-11 09:58:44 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-11 09:58:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:58:49 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-11 09:58:49 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-11 09:58:49 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-11 09:58:49 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_ConnectivityShutdown case ============================
                             [100%]

==================================== ERRORS ====================================
________ ERROR at setup of Test_mtu.test_PI_Networking_Jumbo_Packet_CVL ________

self = <Tests.testNics.test_mtu.Test_mtu object at 0x7f47729d9ac8>

    def setup(self):
        logging.info("this is Test_mtu setup.")
        self.sut_conf = settings.__getattribute__("sut_conf")
        self.pkg_generator = settings.__getattribute__("pkg_generator")
        self.check_str = settings.__getattribute__("check_str")
        self.drive_info = settings.__getattribute__("drive_info")
        self.cn_test_sut = sshInstance()
        logging.info("ssh to the sut1")
        self.cn_test_sut = sshInstance()
        logging.info("ssh to the sut1")
    
        try:
            self.sut1_cli, self.sut1_session = self.cn_test_sut.get_ssh_instance(self.sut_conf["host"], self.sut_conf["user"],
                                                        self.sut_conf["password"],self.sut_conf["port"])
        except:
            logging.info("Case Failed:ssh to sut failed,please check")
            assert 200 == 400, "Case Failed:ssh to sut failed,please check"
    
        try:
            self.sut2_cli, self.sut2_session = self.cn_test_sut.get_ssh_instance(self.pkg_generator["host"], self.pkg_generator["user"],
                                                        self.pkg_generator["password"],self.pkg_generator["port"])
        except:
            logging.info("Case Failed:ssh to sut2 failed,please check")
            assert 200 == 400, "Case Failed:ssh to sut2 failed,please check"
        self.sftp = self.sut1_cli.open_sftp()
>       assert 200 == self.cn_test_sut.modprobe_ice_drive(self.sut1_cli, self.sut_conf["pf1"], self.drive_info["drive_version"])
E       AssertionError: assert 200 == 400
E        +  where 400 = <bound method sshInstance.modprobe_ice_drive of <libs.commons.sshInstance.sshInstance object at 0x7f477270f438>>(<paramiko.client.SSHClient object at 0x7f47764e3208>, 'ens785f2', '1.6.4')
E        +    where <bound method sshInstance.modprobe_ice_drive of <libs.commons.sshInstance.sshInstance object at 0x7f477270f438>> = <libs.commons.sshInstance.sshInstance object at 0x7f477270f438>.modprobe_ice_drive
E        +      where <libs.commons.sshInstance.sshInstance object at 0x7f477270f438> = <Tests.testNics.test_mtu.Test_mtu object at 0x7f47729d9ac8>.cn_test_sut
E        +    and   <paramiko.client.SSHClient object at 0x7f47764e3208> = <Tests.testNics.test_mtu.Test_mtu object at 0x7f47729d9ac8>.sut1_cli

npxtester/Tests/testNics/test_mtu.py:40: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-11 08:53:46 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Jumbo_Packet_CVL_08_53_46/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_PI_Networking_Jumbo_Packet_CVL_08_53_46
2022-04-11 08:53:46 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-11 08:53:46 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-11 08:53:46 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-11 08:53:46 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:53:46 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:53:46 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:53:46 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 08:53:47 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 08:53:47 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 08:53:50 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 08:53:50 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 08:53:55 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 08:53:55 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 08:53:58 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 08:54:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:54:13 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 08:54:13 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 08:54:16 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 08:54:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:54:25 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 08:54:28 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:54:28 - sshInstance.py - [line:54] - root - INFO - ens785f2: ERROR while getting interface flags: No such device

2022-04-11 08:54:36 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 08:54:36 - sshInstance.py - [line:54] - root - INFO - Cannot get driver information: No such device

2022-04-11 08:54:41 - sshInstance.py - [line:209] - root - ERROR - Case Failed:driver version mismatch version ,please check
=================================== FAILURES ===================================
__________ Test_Nics.test_silver_Networking_ConnectivityEnableDisable __________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f4772669c88>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicSilver
    @pytest.mark.nicSimple
    def test_silver_Networking_ConnectivityEnableDisable(self):
        sut1_ip = "192.168.10.10"
        sut2_ip = "192.168.10.11"
        cmd_list = [
            "ping -c 5 {}\n".format(sut1_ip),
            "ifconfig {} down\n".format(self.sut_conf["pf1"]),
            "ifconfig {} up\n".format(self.sut_conf["pf1"]),
        ]
        self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {} {}".format(self.sut_conf["pf1"], sut1_ip))
        self.cn_test_sut.ssh_exec_cmd(self.client2, "ifconfig {} {}".format(self.pkg_generator["pf1"], sut2_ip))
        output, stderr = self.cn_test_sut.run_ssh_cmd(self.session, cmd_list[0], self.check_str)
>       assert 200 == self.cn_test_sut.commond_rs_check(output, "received, 0% packet loss", True)
E       AssertionError: assert 200 == 400
E        +  where 400 = <bound method sshInstance.commond_rs_check of <libs.commons.sshInstance.sshInstance object at 0x7f4772985c18>>('Activate the web console with: systemctl enable --now cockpit.socket\r\n\r\nThis system is not registered to Red Hat ... ping statistics ---\r\n5 packets transmitted, 0 received, 100% packet loss, time 89ms\r\n\r\n[root@npx-cyp7-rh82 ~]# ', 'received, 0% packet loss', True)
E        +    where <bound method sshInstance.commond_rs_check of <libs.commons.sshInstance.sshInstance object at 0x7f4772985c18>> = <libs.commons.sshInstance.sshInstance object at 0x7f4772985c18>.commond_rs_check
E        +      where <libs.commons.sshInstance.sshInstance object at 0x7f4772985c18> = <Tests.testNics.test_nics.Test_Nics object at 0x7f4772669c88>.cn_test_sut

npxtester/Tests/testNics/test_nics.py:114: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-11 08:57:01 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-11 08:57:01 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_silver_Networking_ConnectivityEnableDisable_08_57_01/test_silver_Networking_ConnectivityEnableDisable.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_silver_Networking_ConnectivityEnableDisable_08_57_01
2022-04-11 08:57:01 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_silver_Networking_ConnectivityEnableDisable case ============================
2022-04-11 08:57:01 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 08:57:01 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:57:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 08:57:01 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 08:57:01 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 08:57:01 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 08:57:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:57:04 - sshInstance.py - [line:54] - root - INFO - ens785f2: ERROR while getting interface flags: No such device

2022-04-11 08:57:12 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 08:57:12 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 08:57:17 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 08:57:17 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 08:57:20 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 08:57:20 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 08:57:25 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 08:57:25 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 08:57:28 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 08:57:40 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:57:45 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 08:57:45 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 08:57:48 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 08:57:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:57:57 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 08:58:00 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 08:58:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:08 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 08:58:08 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 08:58:16 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 08:58:16 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:21 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 08:58:24 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 08:58:25 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 08:58:33 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 08:58:33 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095673814  bytes 9228840562340 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851378  bytes 17453491387 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


------------------------------ Captured log setup ------------------------------
INFO     root:NicTestFixture.py:20 test -------
----------------------------- Captured stdout call -----------------------------
2022-04-11 08:58:41 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 192.168.10.10
2022-04-11 08:58:41 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:49 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11
2022-04-11 08:58:49 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 08:58:58 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:09:55 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
2022-04-11 08:59:12 - sshInstance.py - [line:75] - root - INFO - 
--- 192.168.10.10 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 89ms

[root@npx-cyp7-rh82 ~]# 
------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 ifconfig ens785f2 192.168.10.10
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens787f2 192.168.10.11
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:75 Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Mon Apr 11 09:09:55 2022 from 10.112.97.60
[root@npx-cyp7-rh82 ~]# ping -c 5 192.168.10.10
PING 192.168.10.10 (192.168.10.10) 56(84) bytes of data.
INFO     root:sshInstance.py:75 
--- 192.168.10.10 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 89ms

[root@npx-cyp7-rh82 ~]#
_____________ Test_Nics.test_gold_PI_Networking_SRIOV_CheckDevice ______________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f4772664048>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSilver
    @pytest.mark.nicSimple
    @pytest.mark.test
    def test_gold_PI_Networking_SRIOV_CheckDevice(self):
        cmd = "echo 4 > /sys/class/net/{}/device/sriov_numvfs".format(self.sut_conf["pf1"])
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, cmd)
        time.sleep(5)
        if stdout or stderr:
            assert 200 == 400
        sut2_ip = "192.168.10.11"
        self.cn_test_sut.ssh_exec_cmd(self.client2, "ifconfig {} {}\n".format(self.pkg_generator["pf1"], sut2_ip))
        port_up_cmd = "ifconfig {} up"
        manual_ip_cmd = "ifconfig {}{} 192.168.{}.{}"
        ping_ipv4_cmd = "ping -c 5 192.168.{}.{}"
        ping_ipv6_cmd = "ping6 -c 5 -I {}{} {}"
        for i in range(4):
            vf = "v"+str(i)
            ip_part = str(int(sut2_ip.split(".")[3])+i+1)
            time.sleep(5)
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} down".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} up".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, manual_ip_cmd.format(self.sut_conf["pf1"], vf,sut2_ip.split(".")[2] ,ip_part))
            if stdout or stderr:
                assert 200 == 400
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client2, ping_ipv4_cmd.format(sut2_ip.split(".")[2], ip_part))
            time.sleep(3)
            if stdout.find("received, 0% packet loss") == -1:
>               assert 200 == 400
E               assert 200 == 400

npxtester/Tests/testNics/test_nics.py:184: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-11 09:35:44 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-11/test_gold_PI_Networking_SRIOV_CheckDevice_09_35_44/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-11/test_gold_PI_Networking_SRIOV_CheckDevice_09_35_44
2022-04-11 09:35:44 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-11 09:35:44 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-11 09:35:44 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:35:44 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.209
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-11 09:35:44 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-11 09:35:45 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.79
2022-04-11 09:35:45 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-11 09:35:48 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:35:48 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:35:56 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2 | grep ^version:
2022-04-11 09:35:56 - sshInstance.py - [line:54] - root - INFO - version: 1.6.4

2022-04-11 09:36:01 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-11 09:36:01 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-11 09:36:04 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-11 09:36:04 - sshInstance.py - [line:54] - root - INFO - rmmod: ERROR: Module irdma is not currently loaded

2022-04-11 09:36:09 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-11 09:36:09 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-11 09:36:12 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-11 09:36:24 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:29 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-11 09:36:29 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-11 09:36:32 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-11 09:36:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:41 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-11 09:36:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2 up
2022-04-11 09:36:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:36:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens785f2
2022-04-11 09:36:52 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 1.6.4
firmware-version: 3.10 0x80009be5 1.3086.0
expansion-rom-version: 
bus-info: 0000:4b:00.2
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-11 09:37:00 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-11 09:37:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:05 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-11 09:37:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2
2022-04-11 09:37:08 - sshInstance.py - [line:54] - root - INFO - ens785f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe72:ccca  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:72:cc:ca  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 127 (127.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 17  bytes 2314 (2.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-11 09:37:16 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2
2022-04-11 09:37:16 - sshInstance.py - [line:54] - root - INFO - ens787f2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::42a6:b7ff:fe82:eb6  prefixlen 64  scopeid 0x20<link>
        ether 40:a6:b7:82:0e:b6  txqueuelen 1000  (Ethernet)
        RX packets 6095674227  bytes 9228840622489 (8.3 TiB)
        RX errors 1  dropped 1265  overruns 0  frame 1
        TX packets 234851536  bytes 17453517941 (16.2 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


----------------------------- Captured stdout call -----------------------------
2022-04-11 09:37:24 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
2022-04-11 09:37:25 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:38 - sshInstance.py - [line:50] - root - INFO - ifconfig ens787f2 192.168.10.11

2022-04-11 09:37:38 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 down
2022-04-11 09:37:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:37:59 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 up
2022-04-11 09:37:59 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:07 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v0 192.168.10.12
2022-04-11 09:38:07 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:15 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-11 09:38:19 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.405 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.194 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.162 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.207 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.210 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4087ms
rtt min/avg/max/mdev = 0.162/0.235/0.405/0.088 ms

2022-04-11 09:38:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 down
2022-04-11 09:38:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 up
2022-04-11 09:38:43 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f2v1 192.168.10.13
2022-04-11 09:38:51 - sshInstance.py - [line:54] - root - INFO - 
2022-04-11 09:38:59 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.13
2022-04-11 09:39:06 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4110ms
pipe 4

------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 echo 4 > /sys/class/net/ens785f2/device/sriov_numvfs
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens787f2 192.168.10.11

INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 down
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 up
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v0 192.168.10.12
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.12
INFO     root:sshInstance.py:54 PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.405 ms
64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.194 ms
64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.162 ms
64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.207 ms
64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.210 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4087ms
rtt min/avg/max/mdev = 0.162/0.235/0.405/0.088 ms

INFO     root:sshInstance.py:50 ifconfig ens785f2v1 down
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v1 up
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f2v1 192.168.10.13
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.13
INFO     root:sshInstance.py:54 PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
From 192.168.10.11 icmp_seq=1 Destination Host Unreachable
From 192.168.10.11 icmp_seq=2 Destination Host Unreachable
From 192.168.10.11 icmp_seq=3 Destination Host Unreachable
From 192.168.10.11 icmp_seq=4 Destination Host Unreachable
From 192.168.10.11 icmp_seq=5 Destination Host Unreachable

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 0 received, +5 errors, 100% packet loss, time 4110ms
pipe 4
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:88
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:88: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:137
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_iperf_client_4hour.py:137: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:565
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:565: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:691
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:691: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:931
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:931: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1067
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:1067: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1068
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_network_sriov_stress.py:1068: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:102
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:128
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:128: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:158
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:158: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:159
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:159: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:244
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:244: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:270
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:270: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:313
  /root/.jenkins/workspace/cyp-test-nic-simple/npxtester/Tests/testNics/test_nics.py:313: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_simple_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_silver_Networking_ConnectivityEnableDisable
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_gold_PI_Networking_SRIOV_CheckDevice
ERROR npxtester/Tests/testNics/test_mtu.py::Test_mtu::test_PI_Networking_Jumbo_Packet_CVL
= 2 failed, 5 passed, 11 deselected, 18 warnings, 1 error in 3903.18s (1:05:03) =
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4Ix6O+hSs/J+WYhicXoVNFsq2LY7NwaGm8aSejP8W5cDAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRgg5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77aCbP8BPqPtor0AAAA=[0m[Pipeline] }
[8mha:////4GHchb786wl688erDJjASqsmJ9Hr4hPtHO8ytwYHX0MzAAAAoh+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IkMhon14bFL6hQa6F5D9uHMPlF/pr/IJHEyTvdc5bzemOdIo4cnWotdZ5S7VUfBjc/NXLsroFH1fJF1UyJg1WVHStu7GnBisViWZZjpbGxVAdOnpxgq1vzMEUw5IqzxNkdNHLf3PFEpueymChfmCJ2t6FJTL/O3wCmvhfk5V6QlR9NTCl7vQAAAA==[0m[Pipeline] // node
[8mha:////4N3Zwv+i78+NyT5x5Iut/zlc/vu2cqXlCZqUPy9nGyYaAAAAoh+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6IkMhon14bFL6hQa6F5D9uHMPlF/pr/IJHEyTvdc5bzemOdIo4cnWotdZ5S7VUfBjc/NXLsroFH1fJF1UyJg1WVHStu7GnBisViWZZjpbGxVAdOnpxgq1vzMEUw5IqzxNkdNHLf3PFEpueymChfmCJ2t6FJTL/O3wCmvhfkZSnI9h9rVU4xvQAAAA==[0m[Pipeline] End of Pipeline
ERROR: script returned exit code 1
Finished: FAILURE
